scripts/generate-parachain-specs.sh

```diff
@@ -1,31 +1,19 @@
 #!/bin/bash
 source scripts/_init_var.sh
 
-$PARACHAIN_BINARY build-spec \
-  --disable-default-bootnode \
-  | grep '\"code\"' \
-  | head -n1 > $PARACHAIN_SPEC_TMP
-echo $PARACHAIN_SPEC_TMP generated
-
-echo "Using $PARACHAIN_SPEC_TEMPLATE..."
-sed -e "/\"<runtime_code>\"/{r $PARACHAIN_SPEC_TMP" -e 'd;}'  $PARACHAIN_SPEC_TEMPLATE \
-  > $PARACHAIN_SPEC_PLAIN
-echo $PARACHAIN_SPEC_PLAIN generated
```

Comment:
I think the CI (for the release process) was pushing the plain one on the release page. We should fix that

Reply:
I hadn't noticed that the CI was publishing specs at all. I just chatted with @purestakeoskar for a while and we discussed that generating chain specs should not be tied to software releases. Rather it should be tied to launching a new network.

We may release new versions of moonbeam that are only ever put into production via a runtime upgrade, so there should not be any specs associated with them.

Reply:
Concerning the CI, I have few concerns fully removing the specs of the release. It was tied to a release cause it must contain the WASM which is updated when doing a release.
But let's remove it completely from the CI.